src/components/AssetManagementPanel/AssetInformationPanel/index.tsx

```diff
@@ -31,13 +30,10 @@ export const AssetInformationPanel: FunctionComponent<AssetInformationPanel> = s
   }
 )`
   .endorsement-chain-button {
-    color: ${vars.brandBlue};
-    cursor: pointer;
-    transition: color 0.3s ${vars.easeOutCubic};
+    ${tw`text-blue cursor-pointer transition duration-300 ease-out`}
 
```

Comment:
shall we be abit more explicit on transition property? im guessing its only colors so it should be `transition-colors`

Reply:
ok sure